c/datatable.cc

```diff
@@ -172,7 +188,7 @@ void DataTable::delete_all() {
 void DataTable::resize_rows(size_t new_nrows) {
   if (new_nrows == nrows_) return;
   if (new_nrows > nrows_ && nkeys_ > 0) {
-    throw ValueError() << "Cannot increase the number of rows in a keyed frame";
+    throw ValueError() << "Cannot increase number of rows in a keyed frame";
```

Comment:
["cannot increase the number"](https://www.google.com/search?sxsrf=ACYBGNQzQdr2rtmjvarkZfW6_zTmjgLvaA%3A1571677765426&ei=ReatXc_TGc-QtQWAsZBw&q=%22cannot+increase+the+number%22&oq=%22cannot+increase+the+number%22&gs_l=psy-ab.3..0i8i30.5173.5554..5812...0.0..0.98.372.4......0....1..gws-wiz.......0i5i30j0i8i7i30.-_KsUQPg1X0&ved=0ahUKEwiPiqP3663lAhVPSK0KHYAYBA4Q4dUDCAs&uact=5) - 261000 results
["cannot increase number"](https://www.google.com/search?sxsrf=ACYBGNR4Kb8FBbFw8_-w_xa88Pg8vRl2DA%3A1571677772224&ei=TOatXa6bDc_0tAWAjYugDQ&q=%22cannot+increase+number%22&oq=%22cannot+increase+number%22&gs_l=psy-ab.3..0i5i30l2j0i8i30.62745.62858..63189...0.0..0.96.178.2......0....1..gws-wiz.DqYUKJdGBJ4&ved=0ahUKEwiu8cH6663lAhVPOq0KHYDGAtQQ4dUDCAs&uact=5) - 1880 results

Reply:
I will check with a native speaker, because not sure about the context. To me "cannot increase the number of rows" sounds like we cannot increase "this particular number of rows", though we can't increase any number in this case.

Reply:
Looks like "the" in this case is kind of arbitrary, so just leave it as it was before.